plugins/janus_sip.c

```diff
@@ -4502,6 +4503,26 @@ static void *janus_sip_handler(void *data) {
 						gateway->send_pli(session->handle);
 					}
 				}
+			} else if(!strcasecmp(action_text, "pause")) {
+				if(record_audio)
+					janus_recorder_pause(session->arc);
+				if(record_video)
+					janus_recorder_pause(session->vrc);
+				if(record_peer_audio)
+					janus_recorder_pause(session->arc_peer);
+				if(record_peer_video)
+					janus_recorder_pause(session->vrc_peer);
+			} else if(!strcasecmp(action_text, "resume")) {
+				if(record_audio)
+					janus_recorder_resume(session->arc);
+				if(record_video)
+					janus_recorder_resume(session->vrc);
+				if(record_peer_audio)
+					janus_recorder_resume(session->arc_peer);
+				if(record_peer_video)
+					janus_recorder_resume(session->vrc_peer);
+				if(record_video || record_peer_video)
+					gateway->send_pli(session->handle);
```

Comment:
Same here, I guess, even though the property should be part of the SIP plugin session object.